spec/Section 3 -- Type System.md

```diff
@@ -816,11 +817,15 @@ of rules must be adhered to by every Object type in a GraphQL schema.
             the interface field type is either an Interface type or a Union type
             and the object field type is a possible type of the interface field
             type.
-         3. An object field type is a valid sub-type if it is a List type and
+         3. An object field type is a valid sub-type if it is an Interface type
+            and the interface field type is either an Interface type or a Union
```

Comment:
A Union type can't be a subset of an Interface type - I think this needs to be more clear that the interface field type is an implemented Interface type. 

This is kind of a mouthful, I'll take a crack at it here but feel free to reword this and try to make this more clear:

An object field's type is a valid sub-type if it is an Interface type and the interface's field's type is an Interface implemented by the object field's type.

Reply:
I took another look at this, and ended up breaking out the "interface" language from item 2 (so that it now only deals with unions); I updated item 3 so that it deals with interface implementation by objects _and_ interfaces.